site/features.md

```diff
@@ -3,622 +3,262 @@ title: Weave Features
 
-Even when encryption is not in use, certain adverse network conditions
-will cause this fallback to occur dynamically; in these circumstances,
-weave will upgrade the connection back to the fastdp transport without
-user intervention once they abate. You can see which method is in use
-by examining the output of `weave status connections`.
+To use the proxy run: 
 
-You can also administratively disable fastdp with the
-`WEAVE_NO_FASTDP` environment variable:
-
-    $ WEAVE_NO_FASTDP=true weave launch
-
-### <a name="docker"></a>Seamless Docker integration
-
-Weave includes a [Docker API proxy](proxy.html) so that containers
-launched via the Docker
-[command-line interface](https://docs.docker.com/reference/commandline/cli/)
-or
-[remote API](https://docs.docker.com/reference/api/docker_remote_api/)
-are attached to the weave network before they begin execution. To use
-the proxy, run
-
-    $ eval $(weave env)
-
-and then start containers as usual.
-
-Containers started in this way that subsequently restart, either by an
-explicit `docker restart` command or by Docker restart policy, are
-re-attached to the weave network by the weave Docker API proxy.
-
-### <a name="plugin"></a>Docker network plugin
-
-Alternatively, you can use weave as a Docker plugin.  A Docker network
-named `weave` is created by `weave launch`, which you can use like this:
-
-    $ docker run --net=weave -ti ubuntu
-
-> NB: The plugin is an *alternative* to the proxy, hence one should
-> *not* run `eval $(weave env)` beforehand.
-
-For more details see the [plugin documentation](plugin.html).
-
-### <a name="addressing"></a>Address allocation
-
-Containers are automatically allocated an IP address that is unique
-across the weave network. You can see which address was allocated with
-[`weave ps`](troubleshooting.html#list-attached-containers):
-
-    host1$ weave ps a1
-    a7aee7233393 7a:44:d3:11:10:70 10.32.0.2/12
-
-Weave detects when a container has exited and releases its
-automatically allocated addresses so they can be re-used.
-
-See the [Automatic IP Address Management](ipam.html) documentation for
-further details. We also have an explanation of
-[the basics of IP addressing](ip-addresses.html)
-
-Instead of getting weave to allocate IP addresses automatically, it is
-also possible to specify an address and network explicitly, expressed
-in
-[CIDR notation](http://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)
-\- let's see how the first example in the README would have looked:
-
-On $HOST1:
-
-    host1$ docker run -e WEAVE_CIDR=10.2.1.1/24 -ti ubuntu
-    root@7ca0f6ecf59f:/#
-
-And $HOST2:
+    host1$ eval $(weave env)
 
-    host2$ docker run -e WEAVE_CIDR=10.2.1.2/24 -ti ubuntu
-    root@04c4831fafd3:/#
+Containers started in this way that subsequently restart, either
+by an explicit `docker restart` command or by Docker restart 
+policy, are re-attached to the Weave network using the `Weave Docker API Proxy`.
```

Comment:
"using" implies somebody has to do something, but it happens silently in the background.  I think the original "by" was correct.